### PR TITLE
Port of Self-Actualization-Device from BlueMoon

### PR DESCRIPTION
**Pull request by nopeingeneer**
About The Pull Request
Adding S.A.D.

Why It's Good For The Game
Will help those who have DNC order to restore their body genitails even after full body gib. Besides would help to change quirks in round e.t.c.

A Port?
Ported from BlueMoon build

### CODE REVIEW

**Comment by nopeingeneer** [comment]
Hm, i might need advise how to fix integration test

**Comment by BongaTheProto** [comment]
> Hm, i might need advise how to fix integration test

If it's create and destroy don't worry about it, for now

File: tools/UnstandardnessTestForDM/UnstandardnessTestForDM/obj/x86/Debug/UnstandardnessTestForDM.csproj.AssemblyReference.cache
Comment (BongaTheProto) [inline tools/UnstandardnessTestForDM/UnstandardnessTestForDM/obj/x86/Debug/UnstandardnessTestForDM.csproj.AssemblyReference.cache:1]: Could you undo the changes on this file, please?

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

File: tools/UnstandardnessTestForDM/UnstandardnessTestForDM/obj/x86/Debug/UnstandardnessTestForDM.csproj.AssemblyReference.cache
Comment (nopeingeneer) [reply tools/UnstandardnessTestForDM/UnstandardnessTestForDM/obj/x86/Debug/UnstandardnessTestForDM.csproj.AssemblyReference.cache:1]: this is getting out every damn time, just a second

**Comment by nopeingeneer** [comment]
need to re-do this PR due to incorrect execution
